feature-configs/cn-ran-overlays/all-in-one/example/sriov/sriov-network-node-policy-netdevice-mh.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: policy-mh-dpdk-site-1-fqdn-worker1
```

Comment:
Is there a reasoning behind the naming of the files / resources? If so, can you comment it?

The file here is named policy-netdevice although the deviceType is vfio-pci. Would it make sense to rename the file to be more descriptive (i.e. something referring dpdk)?

Reply:
@josephrichard

Reply:
@josephdrichard

Reply:
not aware of any reason for this naming.  I agree it should probably be changed.

Reply:
@josephdrichard, @fedepaol 
Done in 4911f7